AutomatedTesting/Gem/PythonTests/assetpipeline/scene_tests/scene_tests.py

```diff
@@ -0,0 +1,185 @@
+blackbox_scene_tests, blackbox_test_ids = test_builder.parametrize_blackbox_scene_test(dir_test_path)
+
+
+@pytest.mark.usefixtures("asset_processor")
+@pytest.mark.usefixtures("ap_setup_fixture")
+@pytest.mark.usefixtures("local_resources")
+@pytest.mark.parametrize("project", targetProjects)
+@pytest.mark.assetpipeline
+@pytest.mark.SUITE_sandbox
+class TestScene_AllPlatforms(object):
+
+    @pytest.mark.BAT
+    @pytest.mark.parametrize("blackbox_param", blackbox_scene_tests, ids=blackbox_test_ids)
+    def test_SceneBlackboxTest_SourceFiles_Processed_ResultInExpectedProducts(self, workspace, ap_setup_fixture,
+                                                                              asset_processor, project, blackbox_param):
+        """
+        Please see run_fbx_test(...) for details
+        Test Steps:
+        1. Determine if blackbox is set to none
+        2. Run FBX Test
+        3. Determine if override assets is set to none
+        4. If not, re-run FBX test and validate the information in override assets
+        """
+
+        if blackbox_param is None:
+            return
+        self.run_scene_test(workspace, ap_setup_fixture, asset_processor, project, blackbox_param)
+
+        override_assets = blackbox_param.override_assets
+        # If override_assets is set, run the test again and validate the information in the override assets
+        if override_assets is not None:
+            self.run_scene_test(workspace, ap_setup_fixture,
+                                asset_processor, project, blackbox_param, True)
+
+    # Helper: Run Asset Processor with debug output enabled and Atom output disabled.
+    @staticmethod
+    def run_ap_debug_skip_atom_output(asset_processor):
+        result, output = asset_processor.batch_process(capture_output=True, extra_params=["--debugOutput",
+                                                                                          "--regset=\"/O3DE/SceneAPI/AssetImporter/SkipAtomOutput=true\""])
+        # If the test fails, it's helpful to have the output from asset processor in the logs, to track the failure down.
+        logger.info(f"Asset Processor Output: {pformat(output)}")
+        assert result, "Asset Processor Failed"
+
+    # Helper: Compare expected products with products in cache, report on missing products.
+    @staticmethod
+    def check_missing_assets(expected_product_list, cache_folder):
+        missing_assets, _ = utils.compare_assets_with_cache(expected_product_list,
+                                                            cache_folder)
+
+        # If the test is going to fail, print information to help track down the cause of failure.
+        if missing_assets:
+            logger.info(f"The following assets were missing from cache: {pformat(missing_assets)}")
+            in_cache = os.listdir(cache_folder)
+            logger.info(f"The cache {cache_folder} contains this content: {pformat(in_cache)}")
```

Comment:
Let's use one log statement here, or at least group the two statements together.